com.chartboost.mediation.demo/Assets/Demo/Demo.cs

```diff
@@ -193,44 +199,31 @@ public async void OnShowFullscreenClick()
     #endregion
     
     #region Banners
-    private void SetupBannerDelegates()
-    {
-        ChartboostMediation.DidLoadBanner += DidLoadBanner;
-        ChartboostMediation.DidClickBanner += DidClickBanner;
-        ChartboostMediation.DidRecordImpressionBanner += DidRecordImpressionBanner;
-    }
 
-    public void OnCreateBannerClick()
+
+    public async void OnCreateBannerClick()
     {
+        if(_bannerAd != null)
+            Destroy(_bannerAd.gameObject);
+
+        var screenWidthNative = ChartboostMediationConverters.PixelsToNative(Screen.width);
+        
         var size = bannerSizeDropdown.value switch
         {
+            11 => ChartboostMediationBannerAdSize.Adaptive9X16(screenWidthNative),
+            10 => ChartboostMediationBannerAdSize.Adaptive1X4(screenWidthNative),
+            9 => ChartboostMediationBannerAdSize.Adaptive1X3(screenWidthNative),
+            8 => ChartboostMediationBannerAdSize.Adaptive1X2(screenWidthNative),
+            7 => ChartboostMediationBannerAdSize.Adaptive10X1(screenWidthNative),
+            6 => ChartboostMediationBannerAdSize.Adaptive8X1(screenWidthNative),
+            5 => ChartboostMediationBannerAdSize.Adaptive6X1(screenWidthNative),
+            4 => ChartboostMediationBannerAdSize.Adaptive4X1(screenWidthNative),
+            3 => ChartboostMediationBannerAdSize.Adaptive2X1(screenWidthNative),
```

Comment:
Why not just put them in an array and use `bannerSizeDropdown.value` as the index?

Reply:
Done 👍